.github/workflows/test.yml

```diff
@@ -19,17 +19,21 @@ jobs:
         - "test.run.test_noop"
         - "test.run.test_sources"
         - "test.run.test_stages"
-        - "test.src"
+        environment:
```

Comment:
I just wonder: do we need to test against all of these? We currently support these operating systems:
- EL 8 (3.6)
- EL 9 (3.9)
- Fedora 37 (3.11)
- Fedora 38 (3.11)
- Fedora 39 (3.12)

Therefore, I wonder whether we should limit the matrix to just 3.6, 3.9, 3.11, 3.12.

Advantages:
- lower chance of a random failure
- less CO2

Reply:
These are all Python versions that we claim to support in our README right now. We can disable a few of them. Isn't it possible to switch interpreters to any of the other versions on EL?

Reply:
As a side question, do we support operating systems or Python versions with 'osbuild'? Both make sense but I know were also packages for arch and people experiment on Debian and Ubuntu as well.